siemens_cp1616/include/siemens_cp1616/pniobase.h

```diff
@@ -0,0 +1,439 @@
+/****************************************************************************/
```

Comment:
These files don't have a license header.  With just a copyright, the assumption is that they are proprietary and can't be open sourced.  Is this part of the Siemens library with the BSD-like license?